setup.py

```diff
@@ -570,6 +582,13 @@ def srcpath(name=None, suffix=".pyx", subdir="src"):
     include = data.get("include", [])
     include.append(numpy.get_include())
 
+    if name == "io.rdata._rdata" and is_platform_mac():
+        # non-conda builds must adjust paths to libiconv .h and lib dirs
+        include = [
+            os.path.join(os.environ["CONDA_PREFIX"], "include"),
```

Comment:
Should you check if CONDA_PREFIX is defined?  If it isn't, presumably this isn't conda.

Reply:
Thanks for this catch. Turns out this `if` block is not needed. I removed it.